pkg/cliClient/cliClient_test.go

```diff
@@ -342,7 +342,7 @@ func test_getVersionMessage_success() *GetVersionMessageTestCase {
 }
```

Comment:
empty "prerun.json" file
created by accident?

Reply:
because the file is actually a mock of the response of the `prerun` call to the backend